subsys/bluetooth/host/smp.c

```diff
@@ -4453,7 +4453,7 @@ static const struct {
 	{ smp_keypress_notif,      sizeof(struct bt_smp_keypress_notif) },
 };
 
-static int bt_smp_recv(struct bt_l2cap_chan *chan, struct net_buf *buf)
+static int smp_recv(struct bt_l2cap_chan *chan, struct net_buf *buf)
```

Comment:
Unless you plan on updating the other callbacks by removing the `bt_` prefix, I suggest that we keep the `bt_` prefix for now for consistency.